src/lib/alpha.js

```diff
@@ -106,7 +108,7 @@ for (let locale, i = 0; i < farsiLocales.length; i++) {
 }
 
 // Source: https://en.wikipedia.org/wiki/Decimal_mark
-export const dotDecimal = ['ar-EG', 'ar-LB', 'ar-LY'];
+export const dotDecimal = ['ar-EG', 'ar-LB', 'ar-LY', 'fr-CA'];
```

Comment:
According to many sources (even the link in the code comment) francophone area of Canada (fr-CA) are using decimal comma separator. Can you check this?

Reply:
Your're right I made a mistake. I'm in Quebec (canada-french) and most people i work with hate decimals for numbers. I was sure we had the same standard as English.

So yes, in french this is the `decimal` and not the `point`. Sorry